sda/processor_test.go

```diff
@@ -64,7 +65,7 @@ func TestProcessorHost(t *testing.T) {
 		basic, ok := m.Msg.(basicMessage)
 		assert.True(t, ok)
 		assert.Equal(t, basic.Value, 10)
-	default:
+	case <-time.After(100 * time.Millisecond):
```

Comment:
back to (arbitrary) timeouts? Are you experimenting with that or will this stay? If the latter: create a comment why 100ms are reasonable.